src/ServiceControl/Bootstrapper.cs

```diff
@@ -175,6 +176,8 @@ private void RecordStartup(LoggingSettings loggingSettings)
 
             var logger = LogManager.GetLogger(typeof(Bootstrapper));
             logger.Info(startupMessage);
+            endpointConfiguration.SetDiagnosticsPath(loggingSettings.LogPath);
+            endpointConfiguration.GetSettings().AddStartupDiagnosticsSection("Startup", startupMessage);
```

Comment:
I would prefer if we added a diagnostic for each of the items in the startup object. This way they are not just a single output object

Reply:
@WilliamBZA with the approach of having a section for the settings and having a section for the logsettings this would be covered. @WojcikMike and I agreed that the startup message remains as is and will not be used for a section input